### PR TITLE
hello-rails-react

### PR DESCRIPTION
**Pull request by emmanuellekamwa**
In this project I have:
- Created a new Rails app called 'hello-rails-react'.
- Initialized your project with Git
- Created a table for storing the messages and create 5 different greetings
- Created an API endpoint that selects a random greeting from the table

I have an issue with the node modules. I have added the node modules inside the gitignore file but I still see it inside the repo. Please can you give me a suggestion on how to solve it? Thanks

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#dependencies
```
Comment (uwadonat) [inline .gitignore:1]: - [ ] Let's remove node_modules in your repo

- Just go ahead and delete that folder on GitHub first
- Delete the same directory in your local project
- clean your cache by: `npm cache clean --force`
- Add and commit your changes.
- Push again
- Your problem may be resolved.

**Review by uwadonat — CHANGES_REQUESTED** [review]
Hi **[emmanuellekamwa](https://github.com/emmanuellekamwa),** :hand: 

Good job so far! :mechanical_arm:
There are some issues that you still need to work on to go to the next project but you are almost there!

### To highlight! :+1: 
- Your commit and PR messages are very descriptive ✔️


## Status: Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: .gitignore (hunk 1 above)
Comment (emmanuellekamwa) [reply .gitignore:1]: When I run the command it says `npm WARN using --force Recommended protections disabled.`

**Review by Lameck1 — APPROVED** [review]
Hi, @emmanuellekamwa,

Your project is complete!  It's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**